mpu6050/mpu6050.c

```diff
@@ -7,12 +7,11 @@
 
 #include "mpu6050-regs.h"
 
+#define READ_REG_NUM 7
 
 struct mpu6050_data {
 	struct i2c_client *drv_client;
-	int accel_values[3];
-	int gyro_values[3];
-	int temperature;
+	int values[READ_REG_NUM];
```

Comment:
Consider organizing values array and separate fields in a union - it will make code more readable when you'll access them in different way (e.g. formatting output for other interfaces)